packages/@ama-sdk/core/src/plugins/mock-intercept/mock-intercept.request.ts

```diff
@@ -37,6 +37,8 @@ export class MockInterceptRequest implements RequestPlugin {
           ...data,
           method: data.method || 'GET'
         };
+        delete requestOption.api;
```

Comment:
Do we really want to delete it as it will modify the request option for all the plugins? Is there a possibility something else relying on this property could crash (since it is mandatory on v11),
Shouldn't we make a copy or override the api toJSON method?